doc/rtd/conf.py

```diff
@@ -15,6 +11,10 @@
 sys.path.insert(0, os.path.abspath("./"))
 sys.path.insert(0, os.path.abspath("."))
 
+from cloudinit import version
+from cloudinit.config.schema import get_schema
+from cloudinit.handlers.jinja_template import render_jinja_payload
+
```

Comment:
Why this change? It seems to be work for me without it. Am I missing something?

Reply:
This change was required in initial commits during RTD builds because the PYTHONPATH wasn't adequately setup   the way we were originally triggering the doc builds. Since we simplified out RTD builds a bit in iterations of this PRwe may be able to back it out. I'll push a test commit to confirm.

Reply:
Confirmed same build issue due to (pip install on RTD) performs[https://app.readthedocs.org/projects/cloudinit/builds/29295682/].

Reply:
Can we add a comment about that please?

Reply:
done.